Spigot-Server-Patches/0601-Fix-nerfed-slime-when-splitting.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Mon, 24 Aug 2020 08:39:06 -0700
+Subject: [PATCH] Fix nerfed slime when splitting
+
+
+diff --git a/src/main/java/net/minecraft/server/EntitySlime.java b/src/main/java/net/minecraft/server/EntitySlime.java
+index e99fd88118a75f36cb93d02aa7c6029bcffd5f10..1422a927ad6ada9dfcb43be85d634f90ea5ff3b5 100644
+--- a/src/main/java/net/minecraft/server/EntitySlime.java
++++ b/src/main/java/net/minecraft/server/EntitySlime.java
+@@ -205,7 +205,7 @@ public class EntitySlime extends EntityInsentient implements IMonster {
+                 if (this.isPersistent()) {
+                     entityslime.setPersistent();
+                 }
+-
```

Comment:
small nitpick but approving, can we get rid of the line removal

Reply:
done